CHANGELOG.md

```diff
@@ -83,6 +83,10 @@ Read our [guidelines for writing a good changelog entry](https://github.com/biom
 
 ### Formatter
 
+#### New Features
+
+- Added a new option called `--bracket-spacing` to the formatter. This option allows you to control whether spaces are inserted around the brackets of object literals. [#627](https://github.com/rome/tools/issues/627)
+
```

Comment:
very small thing, but you added the changelog line in the already released latest version. it's supposed to be in the "unreleased" tab. also, you are linking to the rome repo instead of this one, making the link point to a random gh issue (instead of this pull request). we also add a `Contributed by @<user-who-added-it>` at the end, in this case you should put your own username in there :)

Reply:
👀 that's what i get for copy pasting multiple times and trying to rebase lol. Thanks for catching this